tests/tests.rs

```diff
@@ -336,7 +336,7 @@ fn cloned_builder() {
     builder.add(String::from("abc")).add(123);
 
     let mut world = World::new();
-    let e = world.spawn(&builder.build().clone());
+    let e = world.spawn(&builder.build());
```

Comment:
This defeats the point of the test.

Reply:
fixed, but forced into an explicit clippy allow, which I find ugly (i like repos to magically pass clippy lints but otherwise not have any obvious signs that they're aware of clippy, since otherwise, of course, you're just trading noise of one kind for noise of another).

Alternative that is just weirder:
```
    let e = world.spawn(&Clone::clone(&builder.build()));
```
and that wouldn't require a clippy allow.

That's less a methodology and more clippy just doesn't catch it, so probably not future proof

Reply:
I think this is a fine place to explicitly allow, since it's a pattern that would be dumb anywhere *but* a test of `Clone`'s correctness.